tests/bucket_manager.py

```diff
@@ -45,13 +45,15 @@ def boto_bucket(
 
 
 @contextmanager
-def boto3_bucket(access_key, secret_key, host, bucket_name=None, **kwargs):
+def boto3_bucket(access_key, secret_key, host=None, bucket_name=None, **kwargs):
     import boto3
 
     name = bucket_name or "testrun-bucket-{}".format(uuid())
-    s3_client = boto3.client("s3")
+    if host == "127.0.0.1":
+        host = "http://127.0.0.1:9000"
```

Comment:
Why is this necessary? Couldn't you directly pass `http://127.0.0.1:9000` to the function call?

Reply:
Was confused about the configuration file, but changing the host there seems to work 👍🏼